src/app/components/mnoe-users-local-list/mnoe-users-local-list.html

```diff
@@ -32,7 +35,14 @@
               <span ng-show="user.isSendingInvite"><i class="fa fa-spinner fa-pulse fa-fw"></i></span>
               {{'mnoe_admin_panel.dashboard.users.widget.local_list.resend_invite' | translate}}
             </button>
-            <span ng-show="user.status === 'active'" translate>mnoe_admin_panel.dashboard.users.widget.local_list.active</span>
+            <button ng-show="!isUserActive(user.status)" class="btn btn-xs btn-warning" ng-disabled="user.isSendingInvite" ng-click="removeUserFromOrganization(user)">
+              {{'mnoe_admin_panel.dashboard.users.widget.local_list.cancel_invite' | translate}}
+            </button>
+            <span ng-show="isUserActive(user.status)" translate>mnoe_admin_panel.dashboard.users.widget.local_list.active</span>
+          </td>
+          <td>
+            <select ng-init="user.roleTlKey=tlKeyFromUserRole(user.role)" ng-model="user.roleTlKey" ng-options="role | translate for role in availableRoles" ng-change="updateUserRole(user)"></select>
```

Comment:
Avoid using `ng-init`, it is a bad practice. This should be done in the controller.